x-pack/tasks/helpers/flags.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { resolve } from 'path';
+
+import log from 'fancy-log';
+import getopts from 'getopts';
+import { toArray } from 'rxjs/operators';
+
+// @ts-ignore
+import { findPluginSpecs } from '../../../src/legacy/plugin_discovery';
+
+/*
```

Comment:
nit: perhaps add a `help` flag to output this?

Reply:
I would, but gulp hijacks the `--help` flag. If we migrate this away from gulp (which would be ideal) then we could definitely do this.